src/utils/url.ts

```diff
@@ -92,13 +92,31 @@ export const openUrl = (url: string, options?: OpenUrlOptions) => {
   }
 };
 
-export const isValidUrl = (url: string): boolean => {
+export const isValidUrl = ({
```

Comment:
let's refactor this! According to our discussion.

Reply:
fixed f0bbdc8